app/sidekiq/central_mail/submit_form4142_job.rb

```diff
@@ -143,7 +143,7 @@ def response_can_be_logged(response)
     end
 
     def processor
-      @processor ||= EVSS::DisabilityCompensationForm::Form4142Processor.new(submission, jid, validate: false)
+      @processor ||= EVSS::DisabilityCompensationForm::Form4142Processor.new(submission, jid)
```

Comment:
I removed the `validate` param being passed here since the base processor class only validates if the corresponding feature flag is enabled anyway.